kraken-std/src/kraken/std/cargo/__init__.py

```diff
@@ -322,6 +324,9 @@ def cargo_build(
         additional_args.append(crate)
     if mode == "release":
         additional_args.append("--release")
+    if features:
+        additional_args.append("--features")
+        additional_args.append(",".join(features))
```

Comment:
```suggestion
        additional_args.extend(features)
```

so that the final command looks like `["--features", "feature1", "feature2"]` rather than `["--features feature1 feature2"]`

Reply:
According to https://doc.rust-lang.org/cargo/reference/features.html , it needs to look like `--features foo-feat,bar-feat`, so the final command needs to look like `["--features", "feature1,feature2"]`.

Reply:
from the [docs](https://docs.python.org/3/library/subprocess.html#frequently-used-arguments)

> Providing a sequence of arguments is generally preferred, as it allows the module to take care of any required escaping and quoting of arguments (e.g. to permit spaces in file names).